### ISSUE TITLE
There is no Add-ons->Code Blocks->Start

### ISSUE THREAD

**Issue by MReed1959** [alexwforsythe/code-blocks#83]
The only thing in the menu is "Help"

**Comment by alexwforsythe** [comment]
Hi @MReed1959. Can you try refreshing your document a few times to see if it starts working? Also, can you check if the add-on works in a new document? Some users have reported that the add-on doesn't show up in certain files for some reason.

**Comment by MReed1959** [comment]
Yes, as soon as I closed the document and re-opened it the 'START' appeared.

Also, I had the document open in another window when I installed the app.

Thanks for such a quick reply!

**Comment by mardi08** [comment]
I am still having the same problem.

there's only 
Add-ons
     -> Get add-ons
     -> Manage add-ons

I have enabled the code block add-ons but it's not showing up. is there a limitation on shared documents?

**Comment by mardi08** [comment]
I found a way.

if you are using chrome, reset the settings

I think this happens due to other extensions that interfere with codeblocks or just mess up the whole settings (I happened to have a lot of extensions).

**Comment by Korb** [comment]
> The only thing in the menu is "Help"

Same.

--- 

Code Blocks February 26, 2022 (latest)
Mozilla Firefox 152.0b8 (64-bit)
Microsoft Windows 10.0.26200.8524

**Comment by bobpaul** [comment]
Disappearance of the start button was caused by deprecation of the old Rhino runtime for apps scripts. See #270

This should be fixed by [pr275](https://github.com/alexwforsythe/code-blocks/pull/275) which was recently merged, though it looks like the addons store still lists Feb 2022 as the last update.

Look at pr275 for instructions to self-host code blocks until it's updated in the addon store. 

As another alternative, @rishi-raj-jain has shared a [pre-formatted block ](https://github.com/alexwforsythe/code-blocks/issues/271#issuecomment-4628918470) you can copy/paste for the time being.

**Comment by alexwforsythe** [comment]
Should be fixed by #275, which is now live
